### PR TITLE
fix: Don't panic on `remove` op

### LINKED ISSUE #204: bug: `REMOVE 'file/in/volume'` panics
It's caused by the following section:
https://github.com/databricks/databricks-sql-go/blob/d70ab7c5b6dda71f1439bd3edc51a2d2022f7d92/connection.go#L579-L586

It's assumed when handling staging requests to always have 4 columns in the return, however, `remove` returns only 3 (there's no local file).

### PR DESCRIPTION
**Pull request by candiduslynx**
Closes #204

### CODE REVIEW

File: connection.go [hunk 1]
```diff
@@ -577,8 +577,8 @@ func (c *conn) execStagingOperation(
 		return dbsqlerrint.NewDriverError(ctx, "error fetching staging operation results", err)
 	}
 	var stringValues []string = make([]string, 4)
-	for i := range stringValues {
-		if s, ok := sqlRow[i].(string); ok {
+	for i, val := range sqlRow { // this will either be 3 (remove op) or 4 (put/get) elements
+		if s, ok := val.(string); ok {
```
Comment (kravets-levko) [inline connection.go:581]: @andrefurlan-db @rcypher-databricks for me this code looks totally fine, but better if one of you take a look. thanks!

**Comment by candiduslynx** [comment]
(force-pushed to signoff)

**Comment by candiduslynx** [comment]
@kravets-levko any chance to speed this review up & release the new SDK version? It's a blocker for us to be able to load data into databricks via Go

**Review by kravets-levko — APPROVED** [review]

**Comment by candiduslynx** [comment]
@kravets-levko also #207 would be nice

**Review by andrefurlan-db — APPROVED** [review]

**Comment by candiduslynx** [comment]
@andrefurlan-db @kravets-levko could you please release a new tag with this & #207 fixes?

**Comment by kravets-levko** [comment]
@candiduslynx sorry for delay - [v1.5.4](https://github.com/databricks/databricks-sql-go/releases/tag/v1.5.4) just published

**Pull request merged**